src/types/newsData.ts

```diff
@@ -4,4 +4,6 @@ export interface NewsData {
 	time_published: string,
 	authors: string,
 	url: string,
+	banner_image: string | null,
+	topics: { topic: string, relevance_score: string }[],
```

Comment:
I don't like nested types, I prefer to do a new interface and use it here
```suggestion
	topics: Topic[],
```